en/docs/SetupAndInstall/UpgradingWSO2APIManager/upgrading-from-250-to-300.md

```diff
@@ -3,350 +3,1678 @@
 The following information describes how to upgrade your API Manager server **from APIM 2.5.0 to 3.0.0** .
 
 !!! note
-To upgrade **from a version older than 1.8.0** , follow the instructions in the document that was released immediately after your current release and upgrade incrementally.
+    To upgrade **from a version older than 1.8.0** , follow the instructions in the document that was released immediately after your current release and upgrade incrementally.
 
-!!! tip
-    Before you begin
+!!! attention "Before you Begin"
```

Comment:
There is  a note prior to this section mentioning "Before you follow this section, see Upgrading Process for more information."

Reply:
Fixed